robot-core/src/main/java/org/obolibrary/robot/ReportOperation.java

```diff
@@ -841,7 +846,7 @@ private static String getQueryResultOrNull(QuerySolution qs, String var) {
    * @return List of Violations
    * @throws IOException on issue parsing query
    */
-  private static List<Violation> getViolations(
+  public static List<Violation> getViolations(
```

Comment:
Ideally better not make these changes unless a separate issue explains why they are needed.

Reply:
Ah I see, you need this for the test to work?

Reply:
yes